src/deckgl/raster-layer/raster-layer.js

```diff
@@ -98,7 +99,8 @@ export default class RasterLayer extends BitmapLayer {
 
   updateState({props, oldProps, changeFlags}) {
     // setup model first
-    if (changeFlags.extensionsChanged) {
+    const modulesChanged = props && props.modules && (oldProps && !isEqual(props.modules, oldProps.modules) || !oldProps);
```

Comment:
Do you need `|| !oldProps`? I would've thought

```suggestion
    const modulesChanged = props && props.modules && oldProps && !isEqual(props.modules, oldProps.modules);
```
would be enough because the `&& oldProps` is already taking care of what `|| !oldProps` would do.